hazelcast-hibernate/hazelcast-hibernate3/src/main/java/com/hazelcast/hibernate/serialization/ExpiryMarker.java

```diff
@@ -33,10 +31,10 @@
  *     <li>It will always return a null value, resulting in a cache miss</li>
  *     <li>It is only replaceable when it is completely expired</li>
  *     <li>It can be marked by multiple transactions at the same time and will not expire until all transactions complete</li>
- *     <li>It should not be expired unless {@link #matches(SoftLock)} is true</li>
+ *     <li>It should not be expired unless {@link #matches(ExpiryMarker) matches} is true</li>
  * </ul>
  */
-public class ExpiryMarker extends Expirable implements SoftLock {
+public class ExpiryMarker extends Expirable implements Serializable {
```

Comment:
`ExpiryMarker ` is implementing `Serializable` interface, because it is being used inside `UpdateEntryProcessor` and `UnlockEntryProcessor`. Since `EntryProcessor` interface extends `Serializable`, all fields inside EntryProcessors must implement `Serializable`. Otherwise our findbugs build fails with `non-transient non-serializable instance field in Class` error.